NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -95,12 +97,18 @@ public enum ServerEventType : int
         /// <summary>
         /// Executed when blood is placed.
         /// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="IPlayer"/> player, <see cref="Vector3"/> position, <see cref="Quaternion"/> rotation.
```

Comment:
"Player" is too generic. It can be either the shooter or the person being shot.

Additionally position and rotation are calculated client-side, not server-side, so they are not known and are different on different clients, so they shouldn't be passed, because even if we calculate them server-side, they won't represent the blood position seen by the clients.

Reply:
I didn't even know there was an option to put the person who is shooting, I thought there was only the referencehub of the owner of the blood.

Reply:
You are right, it's not possible to put the shooter reference hub there.